docs/sphinx_docs/Input_file_formats/Native/05_theories.md

```diff
@@ -44,8 +44,9 @@ precisely, Alt-Ergo implements the second and third layers from the paper "[A
 Three-tier Strategy for Reasoning about Floating-Point Numbers in
 SMT](https://inria.hal.science/hal-01522770)" by Conchon et al.
 
-*Note*: Support for floating-point arithmetic is enabled by default in
-Alt-Ergo since version 2.5.0.  Previous versions required the use of command
+*Note*: Prelude for floating-point arithmetic is a built-in theory in
+Alt-Ergo since version 2.5.0. Use `--enable-theories fpa` to enable it.
+Previous versions required the use of command
```

Comment:
"Prelude for floating-point arithmetic is a built-in theory" doesn't really make sense as a sentence.

What about something like: "Support for floating-point arithmetic is available as a built-in theory since version 2.5.0. It be enabled using flag `--enable-theory fpa`, which will become the default in a future release. Previous versions used the external prelude mechanism and required command line flags `--use-fpa` and `--prelude fpa-theory-2019-10-08-19h00.ae`."